sample/barebones-config.php

```diff
@@ -308,7 +308,7 @@
 
 $section = array(
 	'title'      => esc_html__( 'Text Area', 'your-textdomain-here' ),
-	'desc'       => esc_html__( 'For full documentation on this field, visit: ', 'your-textdomain-here' ) . '<a href="//docs.redux.io/core/fields/textarea/" target="_blank">//docs.redux.io/core/fields/textarea/</a>',
+	'desc'       => esc_html__( 'For full documentation on this field, visit: ', 'your-textdomain-here' ) . '<a href="https://devs.redux.io/core-fields/textarea.html" target="_blank">//docs.redux.io/core-fields/textarea.html</a>',
```

Comment:
This one still says docs

Reply:
<img width="798" alt="Screen Shot 2021-01-22 at 8 42 57 AM" src="https://user-images.githubusercontent.com/1478421/105434322-ea8a2c00-5c8d-11eb-99d6-513446c18574.png">